live-examples/js-examples/object-prototype-tolocalestring.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js">var date1 = new Date(Date.UTC(2012, 11, 20, 3, 0, 0));
+
+console.log(date1.toLocaleString('ar-EG'));
+// expected output: "٢٠‏/١٢‏/٢٠١٢ ٤:٠٠:٠٠ ص"
+
+var obj1 = new Object();
+
+console.log(obj1.toLocaleString('ar-EG'));
+// expected output: "[object Object]"
```

Comment:
I don't understand what this example is showing...

Reply:
Hmmm. Yes - I see your point. I wanted to show how toLocaleString could be applied to objects, keeping the object intact, but I guess it's a bit nebulous.